spec/lib/lead_provider_api_specification_spec.rb

```diff
@@ -18,6 +18,7 @@
         /api/v1/participants/{id}/defer
         /api/v1/participants/{id}/withdraw
         /api/v1/participants/{id}/change-schedule
+        /api/v1/participants/{id}/resume
```

Comment:
Not a big deal, but it might be worth having defer and resume together, since they're more closely related. I'd also have "withdraw" last.